src/mysql.py

```diff
@@ -68,7 +68,8 @@ def main():
             'ROUND(data_free / 1024 / 1024), '
             'ROUND((data_length + index_length), 2) '
             'FROM information_schema.tables '
-            'WHERE table_schema = %s',
+            'WHERE table_type <> "VIEW" '
```

Comment:
We should probably check for `= 'BASE TABLE'`.  There apparently are also `SYSTEM VIEW` entries there.

Reply:
done